FacebookClone/FacebookClone.BLL/Mappers/UserMapper.cs

```diff
@@ -20,8 +20,8 @@ public static UserDTO ToDTO(this User user)
                 Role = (Roles)user.Role,
                 IsBanned = user.IsBanned,
                 IsEmailConfirmed = user.IsEmailConfirmed,
-                ProfileImage = $"http://localhost:5000/{ImageConstants.UserProfileImageFolder}/{SetImagePath(user.Id, user.ProfileImage)}",
-                CoverImage = $"http://localhost:5000/{ImageConstants.UserCoverImageFolder}/{SetImagePath(user.Id, user.CoverImage)}",
+                ProfileImage = $"https://localhost:5001/{ImageConstants.UserProfileImageFolder}/{SetImagePath(user.Id, user.ProfileImage)}",
```

Comment:
Jebiga nisam ocekivao takav problem, ali to bi se inace izdvojilo u appsettings. Da imas vise environment-a moralo bi i onda bi za svaki citao iz appsettings. Mislio sam da se nece menjati jer pokreces lokalno :(

Reply:
Ovo nije komentar da menjas nesto nego cisto da znas sta bi radio da je pravi projekat koji ide na server posle

Reply:
Mislim da je ok sve pa sam approvao, kad vidis ovo mozes merge-ovati